e2e/configurations/metricbeat/beat-xpack.yml

```diff
@@ -0,0 +1,8 @@
+metricbeat.modules:
+- module: beat
+  metricsets:
+    - stats
+    - state
```

Comment:
BTW, starting with Metricbeat 7.8.0, we will no longer need to specify the `metricsets` when `xpack.enabled: true` is set: https://github.com/elastic/beats/pull/17609. Since this PR here is targeting `master`, I think we should remove `metricset`. When we backport to `7.7` and `6.8` (we will need these branches in this repo, BTW, similar to how https://github.com/elastic/elastic-stack-testing has branches corresponding to product repos), we can add them back.

Reply:
We could pair the branches to stack branches too, and update files in consequence

Reply:
That would be great. Not something we need to do just yet, but just a thought for the future when we start seeing differences between branches (like the one I mentioned above).

Reply:
Opened an issue here: https://github.com/elastic/e2e-testing/issues/119